redesign/cs-emf.html

```diff
@@ -107,9 +107,9 @@ <h3 class="b f5">Project Length</h3>
         </p>
         <h3 class="b f5">Implementation Team</h3>
         <p class="f6">
-          Scrum master
-          System Architect & Q&A
-          Mid-level Developer
+          Scrum master<br>
+          System Architect &amp; QA<br>
+          Mid-level Developer<br>
```

Comment:
All of the case study content needs reviewing and this is with me.

In  the meantime, I don't think it makes any  sense to include the 'level' of developer for the website, '2  Developers' should suffice.  (also this 'level' is also incorrect - as I say will have to review all the content).

@Cleop Thinking we just merge this in and I review all the case studies at the end seeing as we don't have permission to publish these on our website yet anyway.

Reply:
I think having the level of developer is useful for a prospective client as it gives them an idea of how much time it took what level of team to create the product